content/hi/docs/contribute/style/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: प्रलेखन शैली अवलोकन
+main_menu: true
+weight: 80
+---
+
+इस खंड के विषय लेखन शैली, सामग्री स्वरूपण, और संगठन, और कुबेरनेट्स प्रलेखन के लिए विशिष्ट ह्यूगो अनुकूलन का उपयोग करना ये सब मार्गदर्शन प्रदान करते हैं।
```

Comment:
`Hugo` is a software/framework, it is better not to localize it. Also this way the sentence sounds better, WDYT, thoughts?
```suggestion
इस खंड के विषय लेखन शैली, सामग्री स्वरूपण, और संगठन, और कुबेरनेट्स प्रलेखन के लिए विशिष्ट Hugo अनुकूलन का उपयोग करने पर मार्गदर्शन प्रदान करते हैं।
```

Reply:
+ 1 to what @Babapool said.